web/components/StrategyTable.tsx

```diff
@@ -106,7 +112,10 @@ export function StrategyTable(props: StrategiesHandler & { network: NetworkName
     .filter((s) => !s.disabled)
     .some((s) => s.selected);
 
-  function openProjectDetails(strategy: StrategyInformation) {
+  function openProjectDetails(strategy: StrategyInformation){
+    // Do not open the project details if the user is trying to select the project
+    if (window.getSelection()?.type === "Range") return;
```

Comment:
Would you mind explaining why we're using `window.getSelection()` here? Not sure I follow. I feel like we can handle this through mouse events instead

Reply:
We discussed, looks good!